content/api/v1/24-filters.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
This file is not really proving any value so it should be removed.

Reply:
@karl-cardenas-coding This file is needed. The UI code based on the file will create the left pane menu for the api paths